libavformat/isom.h

```diff
@@ -111,6 +111,12 @@ typedef struct MOVSbgp {
     unsigned int index;
 } MOVSbgp;
 
+typedef enum {
```

Comment:
I think your original enum was better (if you want to move it here), just needs some comments:

```
typedef enum {	
    MOV_ENC_NONE = 0,	
    MOV_ENC_CENC,	// AES CTR mode full encryption
    MOV_ENC_CBC1,	// AES CBS full encryption
    MOV_ENC_CENS,	// AES CTR mode partial encryption
    MOV_ENC_CBCS,	// AES CBC mode partial encryption
} MOVEncryptionScheme;	

```

Reply:
I switched the strings match https://github.com/FFmpeg/FFmpeg/blob/master/libavformat/movenc.h